File: spec/code_samples/JavaScript/users@{user}@scores/get.js
var FlatApi = require('flat-api');
FlatApi.ApiClient.instance.authentications.OAuth2.accessToken = process.env.FLAT_ACCESS_TOKEN;

// List the scores for the current authenticated user ("me"):
new FlatApi.ScoreApi().getUserScores('me', {}, function (error, data, response) {
  if (error) {
    console.error(error);
  }
  else {
    console.log('Successfully create the document:', data);
  }
});

File: spec/code_samples/Python/users@{user}@scores/get.py
from pprint import pprint
import os

import flat_api
from flat_api.rest import ApiException

flat_api.configuration.access_token = os.environ['FLAT_ACCESS_TOKEN']

print flat_api.configuration.access_token

try:
    # `user` path parameter, `me` for the current user
    pprint(flat_api.UserApi().get_user_scores('me'))
except ApiException as e:
    print e
